src/nidcpower/examples/nidcpower_measure_record_grpc.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import nidcpower
+import sys
+
+
+def example(resource_name, options, voltage, length, address, port):
+    session_name = ''  # user-specified name; empty string means use a new, unnamed session
+
+    # Connect to the grpc server
+    channel = grpc.insecure_channel(f'{address}:{port}')
+    session_options = nidcpower.GrpcSessionOptions(channel, session_name)
+
+    with nidcpower.Session(resource_name=resource_name, options=options, _grpc_options=session_options) as session:
```

Comment:
The existing examples from which you copied are inconsistent in one aspect I just noticed. After the `with` statement that introduces the session, here I see a blank line (as it is in the corresponding local example). The 3 others in the PR you created last week did not have this. Thoughts @marcoskirsch @ni-jfitzger ? Should we remove the extra blank line not only here but also in the existing example?

Reply:
Yeah, that does appear to be a minor inconsistency. I wouldn't reject a cleanup of the NI-DCPower examples to be more consistent about that spacing, but it also doesn't really bother me. I don't think it's the sort of the thing that "black" would even change and it's readable either way.

Reply:
I went ahead and removed the extra blank line from both.

Reply:
Please make the same change to the other examples, including the disabled example, for consistency.

Reply:
Done!